.travis.yml

```diff
@@ -1,9 +1,12 @@
 language: node_js
 node_js:
   - "0.10"
+  - "4.3"
+  - "4.4"
```

Comment:
Version 4.4.2 should be enough rather than 2 versions of node4.
This is the version used in rhscl
```
docker run --rm registry.access.redhat.com/rhscl/nodejs-4-rhel7 node -v
v4.4.2
```